metadata/stream/rationale.yaml

```diff
@@ -51,6 +78,24 @@ architectures:
               signature: https://artifacts.example.com/ais7tah1aa7Ahvei.tar.gz.sig
               sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
               uncompressed-sha256: 38acb15d02d5ac0f2a2789602e9df950c380d2799b4bdb59394e4eeabdd3a662
+      ibmcloud:
+        release: 30.1.2.3
+        formats:
+          "qcow2.xz":
+            disk:
+              location: https://artifacts.example.com/aeng0xah6vaaVosh.qcow2.xz
+              signature: https://artifacts.example.com/aeng0xah6vaaVosh.qcow2.xz.sig
+              sha256: ae3b0c44298fc1ce41e4649b149afbf4c8996fb92427934ca495991b7852b855
+              uncompressed-sha256: 02e9df950c38acb1538d02d5ac0f2a278960d2799b4bdb59394e4eeabdd3a662
+      kubevirt:
+        release: 30.1.2.3
+        formats:
+          "qcow.xz":
```

Comment:
Looks like we have a few existing instances of `qcow` throughout the file that should all be changed to `qcow2`.

Reply:
yes, have changed the `qcow` to `qcow2`